engine/thirdparty/imgui/src/vulkan/glsl_shader.vert

```diff
@@ -0,0 +1,25 @@
+#version 450 core
```

Comment:
in this shader you added  "core", but not in all of them

Reply:
Auto fill in by Rider. Core is for OpenGL core, and ES if we were using OpenGLES, but I can remove this

Reply:
no need, i missed that this was an imgui shader